lib/worker-type.js

```diff
@@ -719,7 +719,10 @@ WorkerType.prototype.determineSpotBids = function (managedRegions, pricing, chan
         for (let zone of zones) {
           try {
             let potentialBid = pricingData[region][type][zone];
-            let potentialPrice = potentialBid / uf[type];
+            let bias = biaser.getBias(region, zone, type);
+            let potentialPrice = potentialBid / uf[type] * bias;
```

Comment:
```
> 2 * 3 / 4
1.5
> (2 * 3) / 4
1.5
> (2 / 3) * 4
2.6666666666666665
> 2 / 3 * 4
2.6666666666666665
```
The linter will complain if you leave in unneeded brackets saying that they're useless, and they are, so I think there's not really much value in grouping them unnecessarily.